app/components/tabs_component.html.erb

```diff
@@ -1,6 +1,7 @@
-<div class="page-tab-container">
-  <% tabs.each do |tab| %>
-    <%= tab %>
-  <% end %>
-</div>
-<hr class="govuk-!-margin-top-0"/>
+<nav class="hmcts-sub-navigation" aria-label="Sub navigation">
+  <ul class="hmcts-sub-navigation__list">
```

Comment:
I don't think we want `hmcts` in our codebase?

Reply:
Spoken with all about using the govuk components classes - he was adamant he wanted the look for sub navigation from hmtc - as an interrim solution - I've renamed the classes and put them in tabs css